app/controllers/api/v1/places_controller.rb

```diff
@@ -1,6 +1,11 @@
 class  Api::V1::PlacesController < ApplicationController
   def index
-    fetch_place_photo = PlaceFacade.new(params[:query]).find_place
+    if params[:search] == "random"
+      location = LocationFacade.new.get_coords(params[:query])
```

Comment:
Will the param `query` always be a zip code? If so, I'd probably change the name to `zip_code`. I wasn't sure what we were expecting here til I got down to your tests. Query is a super vague name.